### PR TITLE
Resolves #1139 : Solves nav-bar bug for admin

### LINKED ISSUE #1139: nav-bar admin bug
**I'm submitting a ...**
- [X] bug report
- [ ] feature request

Nav Bar shows admin option for me although I am not an admin on badgeyay.com

**I am looking into this bug**
admin-bug

### PR DESCRIPTION
**Pull request by gabru-md**
Resolves #1139 

#### Checklist

- [X] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [X] My branch is up-to-date with the Upstream `development` branch.
- [ ] I have added necessary documentation (if appropriate)

#### Changes proposed in this pull request:

- This Pull Request resolves the nav-bar admin bug for now.

An explanation can be found on the related issue :smile: 

Cheers!

### CODE REVIEW

**Comment by gabru-md** [comment]
@yashLadha please take a look :+1:

File: frontend/app/routes/application.js [hunk 1]
```diff
@@ -13,7 +13,7 @@ export default Route.extend({
     if (userObj !== undefined) {
       const uid = this.get('session.uid');
       const adminStatus = localStorage.getItem('adminStatus');
-      if (adminStatus) {
+      if (adminStatus == 'true') {
```
Comment (yashLadha) [inline frontend/app/routes/application.js]: Can we use strict type checking using `===`

File: frontend/app/serializers/user.js [hunk 2]
```diff
@@ -11,7 +11,7 @@ export default JSONAPISerializer.extend({
   },
 
   normalizeResponse(store, primaryModelClass, payload, id, requestType) {
-    if (payload.data.attributes.siteAdmin) {
+    if (payload.data.attributes.siteAdmin == 'true') {
       this.authToken.enableAdmin();
```
Comment (yashLadha) [inline frontend/app/serializers/user.js]: Here too

**Comment by yashLadha** [comment]
@gabru-md  Can you please squash your commits? So that I can merge it

**Pull request merged**